include/SDL3/SDL_events.h

```diff
@@ -615,7 +620,15 @@ typedef struct SDL_GamepadButtonEvent
 /**
  * Gamepad device event structure (event.gdevice.*)
  *
+ * Joysticks that are supported gamepads receive both an SDL_JoyDeviceEvent and
+ * an SDL_GamepadDeviceEvent.
+ *
+ * SDL will send GAMEPAD_ADDED events for joysticks
+ * that are already plugged in during SDL_Init and are recognized as gamepads. It will also send events for joysticks that get gamepad mappings at runtime.
```

Comment:
```suggestion
 * that are already plugged in during SDL_Init() and are recognized as gamepads. It will also send events for joysticks that get gamepad mappings at runtime.
```